### PR TITLE
Reminders UI

### PR DESCRIPTION
**Pull request by SmittieC**
For #93 

I added a model that holds the details required for the Periodic task thats used to create "scheduled messages". Currenly the datetime widget doesn't look very good. I'm looking into adding one from Tailwind that looks better

### Light mode
image

### Dark mode
image

### CODE REVIEW

**Review by snopoke — COMMENTED** [review]
No major comments.

Since these are 'per experiment' it would be nice to be able to create them within the experiment UI instead of having to do it separately.

Related, whats the workflow for these? Do they get created at the start or during the experiment?

File: apps/experiments/models.py [hunk 1]
```diff
@@ -439,3 +441,74 @@ def is_complete(self):
...
+            "name": f"{self.team.slug}-{self.name}",
+            "task": self.SCHEDULED_MESSAGE_TASK,
+            "clocked": clocked,
+            "kwargs": json.dumps(task_kwargs),
+            "one_off": True,
+        }
+        if self.periodic_task:
+            PeriodicTask.objects.filter(id=self.periodic_task.id).update(**periodic_task_kwargs)
+        else:
+            self.periodic_task = PeriodicTask.objects.create(**periodic_task_kwargs)
```
[the remark this answers is not in the document]
Comment (SmittieC) [reply apps/experiments/models.py:496]: Good point! If we don't, the user will see these scheduled messages and will have to remove them by default. I think we can clean it up when it's completed

File: apps/experiments/models.py [hunk 2]
```diff
@@ -439,3 +441,74 @@ def is_complete(self):
...
+            "sent to the bot as an instruction. Set to False if the message should be sent directly to users"
+        ),
+    )
+    message = models.TextField(
+        blank=True,
+        default="",
+        help_text="The message",
+    )
+    clocked_schedule = models.DateTimeField(null=False, blank=False)
+    periodic_task = models.ForeignKey(PeriodicTask, null=True, on_delete=models.CASCADE)
```
[the remark this answers is not in the document]
Comment (SmittieC) [reply apps/experiments/models.py:471]: My reasoning for not doing that was the fact that the `ScheduledMessage` is simply a sort of "proxy" for the `PeriodicTask` model (and `ClockedSchedule` technically), so if we somehow remove the underlying periodic task (which only superusers can do), we should remove the linked `ScheduledMessage` as well. (this reminds me, I should test this behaviour).

**Comment by SmittieC** [comment]
FYI: Keeping this in draft for now. I reached out to Delta to gather some info on how this is currently being used to inform the correct path forward

**Pull request closed without merging**